app/lib/logic/rolling_updates.rb

```diff
@@ -299,16 +299,6 @@ def enabled?
         end
       end
 
-      class ApiAsProduct < Base
```

Comment:
we can also remove this RU, i think:
https://github.com/3scale/porta/blob/183bab3a90bbdf747297a9ec4a6183f0ca199ffe/app/lib/logic/rolling_updates.rb#L132-L136

Reply:
#2274